### ISSUE TITLE
Показывать сниппеты включения/отключения диагностики

### ISSUE THREAD

**Issue by andrewzola** [otymko/phoenixbsl#98]
было бы удобно, если вместе с диагностикой была бы возможность скопировать в буфер обмена  сниппет включения/выключения

**Comment by nixel2007** [comment]
@otymko может подвязать это на планируемый code action в блс лс на добавление игнорирования?

**Comment by andrewzola** [comment]
это было бы еще круче
примерно такого вида:
// bslls диагностика выкл.
// ... тут должен быть комментарий, если есть возможность, то фокус перемещается в начало этой строки
// bslls диагностика вкл.

**Comment by otymko** [comment]
@nixel2007 да, так и сделаем. Я как раз скоро планирую Quick Fix вынести в список замечаний, там и экранирование рядом
